share/qtcreator/sfdk/modules/30-building-builtin.json

```diff
@@ -15,9 +15,9 @@
     [
         {
             "name": "engine",
-            "trSynopsis": "{start|stop|status}",
+            "trSynopsis": "{start|stop|status}\nexec [<command> [<arg>...]]",
             "trBrief": "Control the build engine",
-            "trDescription": "The 'start' and 'stop' commands allow to control the build engine virtual machine conveniently. Starting the build engine explicitly is usually not required - commands that need build engine running will ensure it is running. The 'status' command prints status information."
+            "trDescription": "The 'start' and 'stop' commands allow to control the build engine virtual machine conveniently. Starting the build engine explicitly is usually not required - commands that need build engine running will ensure it is running. The 'status' command prints status information.\n\nThe 'exec' command allows executing an arbitrary <command> inside the build engine. If no <command> is specified, an interactive shell is executed."
```

Comment:
I think it could be mentioned that path mapping is performed on the output when running the exec command. It can be a bit surprising - e.g. try "sfdk engine exec echo /home/mersdk/share"

Reply:
Checked that, it would need to be mentioned under most of the other commands as well, so I think the paragraph explaining path mapping under domain help should be enough.